include/seqan3/core/platform.hpp

```diff
@@ -269,7 +269,7 @@
 //!\brief See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=96070 and https://github.com/seqan/product_backlog/issues/151
 #ifndef SEQAN3_WORKAROUND_GCC_96070 // not yet fixed, this is a defect within the standard lib
 // remind us of this issue once gcc-11 was released.
-#   if defined(__GNUC__) && (__GNUC__ < 11)
```

Comment:
@rrahn I'm unhappy that we bound this to a specific version. You are right that it reminds us of re-checking it, but this has the down-side that we purposely break our code for future gcc versions for our users. Imagine someone doe not want to transition from 3.0.2 to 3.0.3 immediately, because it breaks some interfaces. It is weird to say to our users, hey we upstream developers need to check this.

I would prefer to add a release-task in our check-list to see if the WORKAROUNDS within platform.hpp were fixed in the meantime. This might not notify us immediately in a new gcc-version, but it would be part of our process.

Reply:
Mhmm, I have mixed feelings about this. You are right that the transition to a newer compiler might be less trouble some with an older SeqAn version. But on the other hand that's the beauty of the versioning. The only thing we say is that version 3.0.2 has no official support for gcc-11. So people either upgrade to the new SeqAn version or they work with an older compiler. I mean there is no guarantee that the workaround itself might not break as well with future compiler versions (also less likely but we have seen quite the regressions already). So all I am saying is, SeqAn 3.0.2 officially supports up to gcc-10. Starting with 3.0.3 we support gcc-11 as well, also if this means just using the same workaround, but we don't get into issues like reporting bugs upstream too late, having not enough time to fix parts that could have been resolved before the release etc.

Reply:
How about a `SEQAN3_WORKAROUND_TEST_UNRESOLVED_ONES` xD

Reply:
Hmm.. I don't have a clear opinion on this topic. we should probably discuss this in the DEV meeting if you want to